Gemfile

```diff
@@ -3,9 +3,9 @@ source "https://rubygems.org"
 gemspec
 
 gem "rake"
-gem "redis-namespace", github: "resque/redis-namespace", branch: :master
+gem "redis-namespace", "1.11.0"
 gem "rails", ">= 6.0.2"
-gem "sqlite3", platforms: :ruby
+gem "sqlite3", "~> 1.7", platforms: :ruby
```

Comment:
This allows `gemtester` to work, so it seems good to do here (the Gemfile gets used for CI/tests only)